functions.php

```diff
@@ -26,28 +26,49 @@ function displayChairs($query): string
         $result .= '<img src="' . $chair['imagelink'] . '" height="250"  alt="A lovely chair" /><br>';
         $result .= '<span>Designer: ' . $chair['designer'] . '</span><br/>';
         $result .= '<span>Year: ' . $chair['designyear'] . '</span><br/></div>';
+        $result .= '<form method="post" action="index.php"><input name="delete" type="submit" value="Remove this chair from the collection" /><input type="hidden" name="id" value="' . $chair['id'] . '" /></form>';
     }
     return $result;
 }
 
 /** create a function to insert the POST form data into the database
+ * @param PDO $db
 
-function insertChair()
-{
-    if(isset($_POST['submit']))
+ */
+
+function insertChair(PDO $db){
+
+    if(isset($_POST['submit'])
+        && (!empty($_POST['chairname'])) && (!empty($_POST['designer'])) && (!empty($_POST['designyear'])) && (!empty($_POST['imagelink'])))
     {
-    $chairname = $_POST['chairname'];
-    $designer = $_POST['designer'];
-    $designyear = $_POST['designyear'];
-    $imagelink = $_POST['imagelink'];
-
-    $insert = $db->prepare("INSERT INTO `chairs` (`chairname`, `designer`, `designyear`, `imagelink`) VALUE (:chairname, :designer, :designyear, :imagelink);
-    $insert->bindParam(':chairname', $chairname);
-    $insert->bindParam(':designer', $designer);
-    $insert->bindParam(':designyear', $designyear);
-    $insert->bindParam(':imageline', $imagelink);
-    $insert->execute();
+        $chairname = $_POST['chairname'];
+        $designer = $_POST['designer'];
+        $designyear = $_POST['designyear'];
+        $imagelink = $_POST['imagelink'];
+
+        // $db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+
+        $insertchair = $db->prepare("INSERT INTO `chaircollection` (`chairname`, `designer`, `designyear`, `imagelink`, `deleted`) VALUE (:chairname, :designer, :designyear, :imagelink, 0)");
+    $insertchair->bindParam(':chairname', $chairname);
+    $insertchair->bindParam(':designer', $designer);
+    $insertchair->bindParam(':designyear', $designyear);
+    $insertchair->bindParam(':imagelink', $imagelink);
+    $insertchair->execute();
```

Comment:
You can add the following or equivalent for better UX after the execute action:
header('Location: index.php');
    exit;